test/onnx/gen_onnx.py

```diff
@@ -1315,6 +1315,22 @@ def conv_bias_test():
     return ([node], [x, y, z], [out])
 
 
+@onnx_test()
+def conv_bad_bias_test():
```

Comment:
Not sure I follow the negative test here. Is it the wrong type of tensors x & y? Thanks.

Reply:
This was added as I'm modifying parse_convolution which handles both quantized convolution and regular convolution. 

I noticed we were missing a type check on the bias used for convolution and added the test case. In short, bias arguments must be the identical type to which input they apply too. We weren't covering that case. Here Z, is int32 instead of FLOAT like the X input which should be an error.